src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -1398,7 +1459,9 @@ std::vector<DWRITE_FONT_AXIS_VALUE> CustomTextLayout::GetAxisVector(DWRITE_FONT_
                                         &mappedFont,
                                         &scale);
 
-                RETURN_IF_FAILED(_SetMappedFont(textPosition, mappedLength, mappedFont.Get(), scale));
+                ::Microsoft::WRL::ComPtr<IDWriteFontFace> face;
+                RETURN_IF_FAILED(mappedFont.Get()->CreateFontFace(&face));

```

Comment:
Probably need to check for a null deref here. The return code of `MapCharacters` isn't checked, so I don't think you can be guaranteed that `mappedFont` is appropriately filled before you do the `->`